egs2/opencpop/svs1/conf/tuning/train_visinger_hubert_cn.yaml

```diff
@@ -0,0 +1,210 @@
+            out_channels: 1
+            kernel_sizes: [5, 3]
+            channels: 32
+            downsample_scales: [3, 3, 3, 3, 1]
+            max_downsample_channels: 1024
+            bias: True
+            nonlinear_activation: "LeakyReLU"
+            nonlinear_activation_params:
+                negative_slope: 0.1
+            use_weight_norm: True
+            use_spectral_norm: False
+        multi_freq_disc_params:
+            hop_length_factors: [2.5, 5, 7.5, 10, 12.5, 15]
+            hidden_channels: [256, 256, 256, 256, 256]
+            domain: "double"
+            mel_scale: True
+            divisors: [32, 16, 8, 4, 2, 1, 1]
+            strides: [1, 2, 1, 2, 1, 2, 1]
+
+    # loss function related
+    generator_adv_loss_params:
+        average_by_discriminators: false # whether to average loss value by #discriminators
+        loss_type: mse                   # loss type, "mse" or "hinge"
+    discriminator_adv_loss_params:
+        average_by_discriminators: false # whether to average loss value by #discriminators
+        loss_type: mse                   # loss type, "mse" or "hinge"
+    feat_match_loss_params:
+        average_by_discriminators: false # whether to average loss value by #discriminators
+        average_by_layers: false         # whether to average loss value by #layers of each discriminator
+        include_final_outputs: true      # whether to include final outputs for loss calculation
+    mel_loss_params:
+        fs: 24000          # must be the same as the training data
+        n_fft: 2048        # fft points
+        hop_length: 480    # hop size
+        win_length: 2048   # window length
+        window: hann       # window type
+        n_mels: 80         # number of Mel basis
+        fmin: 0            # minimum frequency for Mel basis
+        fmax: 12000        # maximum frequency for Mel basis
+        log_base: null     # null represent natural log
+    lambda_adv: 1.0        # loss scaling coefficient for adversarial loss
+    lambda_mel: 45.0       # loss scaling coefficient for Mel loss
+    lambda_feat_match: 2.0 # loss scaling coefficient for feat match loss
+    lambda_dur: 0.1        # loss scaling coefficient for duration loss
+    lambda_pitch: 10.0     # loss scaling coefficient for pitch loss
+    lambda_phoneme: 1.0    # loss scaling coefficient for ctc loss
+    lambda_kl: 1.0         # loss scaling coefficient for KL divergence loss
+    # others
+    sampling_rate: 24000          # needed in the inference for saving wav
+    cache_generator_outputs: true # whether to cache generator outputs in the training
+
+# extra module for additional inputs
+pitch_extract: dio         # pitch extractor type
+pitch_extract_conf:
+    use_token_averaged_f0: false
+    use_log_f0: false
+pitch_normalize: None      # normalizer for the pitch feature
+
+# ying_extract: ying
+# frontend related
+frontend: s3prl
+frontend_conf:
+   frontend_conf:
+        upstream: hf_hubert_custom
```

Comment:
Since the change is small, do we need that many configs? (One possible way is to add some options here with comments.

Reply:
Sure, let's keep one hubert config and one fused config.